.github/workflows/unit-tests.yaml

```
@@ -0,0 +1,54 @@
+        - name: Detect Aztec version
+          id: aztec-version
+          run: |
+            AZTEC_VERSION=$(node -p "require('./package.json').config.aztecVersion")
+            echo "version=$AZTEC_VERSION" >> "$GITHUB_OUTPUT"
+            echo "Aztec version is ${{ steps.aztec-version.outputs.version }}"
+
+        - name: Install Aztec CLI
+          run: |
+            curl -s https://install.aztec.network > tmp.sh
+            bash tmp.sh <<< yes "yes"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)